### DISCUSSION TITLE
3.1.2

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-core#5173] [Design & tech]
## What's Changed
* Remove unused BulkExtensions by @Suchiman in https://github.com/elsa-workflows/elsa-core/pull/5160
* Use PolySharp for automatic polyfilling by @Suchiman in https://github.com/elsa-workflows/elsa-core/pull/5161

**Full Changelog**: https://github.com/elsa-workflows/elsa-core/compare/3.1.1...3.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-core/releases/tag/3.1.2'>3.1.2</a>.</em>

**Comment by glime-ai** [comment]
The changes in the Elsa Workflows from version 3.1.1 to 3.1.2 include the removal of unused BulkExtensions and the use of PolySharp for automatic polyfilling. These changes were made in two separate pull requests:

1. The removal of unused BulkExtensions was done by @Suchiman in pull request [#5160](https://github.com/elsa-workflows/elsa-core/pull/5160).
2. The use of PolySharp for automatic polyfilling was also done by @Suchiman in pull request [#5161](https://github.com/elsa-workflows/elsa-core/pull/5161).

For a full list of changes, you can view the full changelog by comparing the tags on GitHub: [3.1.1...3.1.2](https://github.com/elsa-workflows/elsa-core/compare/3.1.1...3.1.2).

**Comment by Suchiman** [comment]
Awesome, now that went quicker than expected, thank you very much 💯
